session9/src/com/itscool/homework/CpmparingEx4.java

```diff
@@ -0,0 +1,22 @@
+package com.itscool.homework;
+
+import java.util.Scanner;
+
+public class CpmparingEx4 {
+    public static void main(String[] args) {
+        Scanner scanner =  new Scanner(System.in);
+
+        System.out.println("Enter first (YYYY-MM-DD): ");
+        String date1 = scanner.nextLine();
+
+        System.out.println("Enter the second time (YYYY-MM-DD): ");
+        String date2 = scanner.nextLine();
+
+        boolean datesEqual = areDatesEqual(date1, date2);
+        System.out.println("Are dates equal? Result: " + datesEqual);
+    }
+
+    public static boolean areDatesEqual(String date1, String date2) {
+        return date1.equals(date2);
```

Comment:
This will work, but here you are comparing plain Strings. Could you convert these 2 strings to `LocalDate` objects and compare them?
Hint: 
1. use utility method `parse` from LocalDate to convert/parse from String to LocalDate objects (for both `date1` and `date2`)
2. apply `isEqual` method on `LocalDate` object obtained above, giving as argument the second `LocalDate` object.